src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,52 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String carBrand;
+    private String model;
+    private String number;
+    private int yearOfCarProduction;
+    private int fuelCapacity;
+    private Engine carEngine;
```

Comment:
let's create getters and setters for all fields and move them before toString() and clone() method

Reply:
done